docker/configs/notification-rules.json

```diff
@@ -0,0 +1,13 @@
+[
+    {
+      "channelName": "channel-1",
+      "channelId": "channel-id-1",
+      "channelConfig": [
+        {
+          "channelConfigType": "WEBHOOK",
+          "url": "https://hooks.slack.com/services/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",
```

Comment:
Remove this url and let's add instructions in docker-compose README about adding this.

Reply:
Removed, will add more details on these later in the docs